lib/friends/activity.rb

```diff
@@ -34,9 +34,25 @@ def initialize(str: "")
       # Partition lets us parse "Today" and "Today: I awoke." identically.
       date_s, _, description = str.partition(DATE_PARTITION)
 
-      # rubocop:disable Lint/AssignmentInCondition
-      if time = (date_s =~ /^\d{4}-\d{2}-\d{2}$/ ? Time : Chronic).parse(date_s)
-        # rubocop:enable Lint/AssignmentInCondition
+      time = if date_s =~ /^\d{4}-\d{2}-\d{2}$/
```

Comment:
I would make this a simple `if` block. Like:
```ruby
if date_s ...
    time = Time.parse(date_s)
else
    past_time = Chronic.parse(date_s, content: :past)
end
```

Then down below, have something like
```ruby
if past_time
  ...
else if time
  ...
else
 ...
end
```

To me, that reads better.